apps/sr-frontend/src/app/App.tsx

```diff
@@ -29,7 +35,16 @@ export default function App() {
               <Stack gap={1} pt={1}>
                 <CharacterSelector />
                 <Character />
-                <LightConeEditor />
+                <Container>
```

Comment:
Move everything in the `<Container>` to its own component, into another file, to keep App.tsx clean.

Reply:
Done! Hopefully now it's good. Please let me know if there are other changes I should make.

Reply:
I was hoping to keep the accordion outside of the component, so it would be less refactor to move this component to a new page. But no worries, should just be a few line change later down the line

Reply:
Okk, I will look into it then